spec/index.bs

```diff
@@ -3503,7 +3503,23 @@ dictionary GPUBufferBindingLayout {
 
     : <dfn>minBindingSize</dfn>
     ::
-        May be used to indicate the minimum buffer binding size.
+        Indicates the minimum buffer binding size.
+        Bindings are validated against this size in {{GPUDevice/createBindGroup()}}.
+
+        If this *is not* `0`, pipeline creation [$validating shader binding|validates$]
```

Comment:
both pipeline creation and bind group creation would error in this case, not just the former

Reply:
I think it already says this, but clarified to be explicit.